test/fields/int.spec.ts

```diff
@@ -194,6 +194,50 @@ describe('Fields', () => {
         },
       })).rejects.toThrow('The following field is invalid: decimalMax');
     });
+
+    it('should validate an array of numbers', async () => {
```

Comment:
Move validation tests to `src/field/validations.spec.ts` since they don't need to be integration.